packages/client/components/ResourceTable/ResourceTable.js

```diff
@@ -130,6 +134,7 @@ class ResourceTable extends React.Component {
                                                 checked={selected.includes(
                                                     row.key
                                                 )}
+                                                data-testid="row-checkbox"
```

Comment:
I was thinking you'd want the data-testid in a loop to be unique ? something like 

```
data-testid={`row-checkbox-${index}`}
```
?